deploy/template.yaml

```diff
@@ -3373,6 +3389,32 @@ Resources:
         - !Ref PermissionsBoundary
         - !Ref AWS::NoValue
 
+  YotiCallbackStateMachineFailedCanary:
+    Type: AWS::CloudWatch::Alarm
```

Comment:
also could set this alarm conditionally so this alarm only deploys if the canary deployment strategy is not all at once
Condition: UseCanaryDeploymentAlarms